docs/DeveloperGuide.adoc

```diff
@@ -133,8 +133,19 @@ The `Model`,
 * Model is the bridge between Logic and Storage and provides an abstraction of how the data is stored in memory.
 * It exposes a `ReadableEntityList` which only has the list method to remind Logic that the data given should not be modified.
 * The UI can be bound to these lists so that it automatically updates when the contents of the list change.
+* At the heart of the model are observable lists which allow for the dynamic updating of the UI.
 * The `Model` interface also serves as an API through which controller can edit the data stored in memory.
 
+`ModelManager`
+* ModelManager implements all the methods exposed by the Model Interface. At its heart, it only contains 3 `EntityLists`
+, 3 `FilteredLists` and a `UserPrefs` Object and does all the validation logic needed for the application
```

Comment:
This will be changed in v1.4 right? Or are we following this format? :)

Reply:
hmmm it was already implemented and is quite a complex class, so i just wrote first if it is ok.